src/package.cmd.js

```diff
@@ -12,14 +12,15 @@
 const chalk = require('chalk');
 const glob = require('glob');
 const path = require('path');
-const fs = require('fs-extra');
+const fs = require('fs');
 const ProgressBar = require('progress');
-const archiver = require('archiver');
-const StaticCommand = require('./static.cmd.js');
+const { ZipFile } = require('yazl');
+const AbstractCommand = require('./abstract.cmd.js');
 const ActionBundler = require('./parcel/ActionBundler.js');
-const { flattenDependencies } = require('./packager-utils.js');
+const { multiline } = require('@adobe/helix-shared').string;
+const { map, join, find, contains } = require('ferrum');
 
-class PackageCommand extends StaticCommand {
+class PackageCommand extends AbstractCommand {
```

Comment:
@koraa Please put the static action removal in its own issue and pull request. I think it makes sense to only use the `helix-services/static` in general. but we should discuss this separately.